src/auth/views/Login.tsx

```diff
@@ -70,7 +77,8 @@ const LoginView: React.FC<LoginViewProps> = ({ params }) => {
       state
     });
     if (result && !result?.errors?.length) {
-      navigate(APP_DEFAULT_URI);
+      navigate(fallbackUri ?? APP_DEFAULT_URI);
```

Comment:
When using `react-router` navigate do not include the `APP_DEFAULT_URI` as it's treated as default url basename. You can see it here:
```
<BrowserRouter basename={APP_MOUNT_URI}>
```